README.md

```diff
@@ -1 +1,63 @@
-# my_blog_app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My_Blog_App
+
+> Ruby on Rails basic project to build a structure for displaying users, their posts, and their comments.
+
+## Run the project
+
+`rails s`
+
+## Built With
+
+- Ruby
+- Ruby linters
+- stylint linters
+- Ruby on Rails
+
+## preequests
+
+- postgresql database in your machine
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- SSH:
+  - `git@github.com:Paskab012/my_blog_app.git`
+- HTTPS:
+  - `https://github.com/Paskab012/my_blog_app.git`
+
+then run:
+
+- `bundle install`
+- `npm install`
+
+## Run tests
+
+Run `rspec`
+
+## Author
+
+👤 **Pascal Kabika Mp**
+
+- GitHub: [Paskab012](https://github.com/KABIKA681?tab=overview&from=2021-12-01&to=2021-12-31)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/pascal-kabika-443061220/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/vikipretium/blog-app/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse: [microverse community](https://github.com/microverseinc)
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
[OPTIONAL] Kindly add MIT license to your project.

Reply:
Thanks @Rustamxon7 , I will take care of adding it.